src/components/home-components/ArticleContainer.jsx

```diff
@@ -0,0 +1,29 @@
+import ArticleCard from './ArticleCard';
+import { api } from '../../api';
+import { useEffect, useState } from 'react';
+
+const ArticleContainer = () => {
+	const [articles, setArticles] = useState([]);
+
+	useEffect(() => {
```

Comment:
We should have some kind of loading state here and be conditionally rendering a loading message to users with a slower connection